krm/gitlab-krm-operator/README.md

```diff
@@ -0,0 +1,89 @@
+# Gitlab KRM Operator
+
+  Gitlab KRM Operator is a kubernetes extension that implements Kubernetes style APIs for Gitlab resources.
+  Currently it supports following APIs:
+
+  - Groups
+  - Projects
+  - DeployKeys
+
+# Prerequisites
+
+  Before you begin installing this operator in a kubernetes cluster, you will need the following:
+
+  - Kubernetes (v1.14+) cluster
+  - Kustomize
+  - Kubectl configured to talk to the kubernetes cluster
+  - Gitlab Base API URL
+  - Access Token to access the APIs
```

Comment:
to access the "GitLab" APIs

Does the token need to have some specific permission/scope?